commitlint.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
+    rules: {
+        // 自定义规则
+        'body-leading-blank': [1, 'always'], // 确保body开始有一个空行
+        'footer-leading-blank': [1, 'always'], // 确保footer开始有一个空行
+        // 其他规则...
+    }
+}
```

Comment:
Use Wizardoc commit convention rather than conventional-commit, please refer https://github.com/wizardoc/commitlint-wizardoc?tab=readme-ov-file#configuration